CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## 88 Release Notes
 
+### New feature: Prevent Job start during Cloud Quota exhaustion
+
+This optional feature prevents Cromwell from starting new jobs in a billing project that is currently experiencing 
```

Comment:
Should we say "in a hog group" here?

Reply:
I can replace `billing project` with `hog group` 👍

Reply:
I'm of the [belief](https://broadworkbench.atlassian.net/browse/WX-1745) that we should not use "hog group" in any new, external-facing places.

Reply:
What do you think would be clearer? I don't think "billing project" is either accurate in Terra or useful to other users.

Reply:
Maybe just `group` might work?

Reply:
Yeah I usually just use group.